installing/installing_aws/installing-aws-user-infra.adoc

```diff
@@ -114,6 +114,8 @@ include::modules/installation-operators-config.adoc[leveloffset=+1]
 
 include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
 
+You can configure registry storage for user-provisioned infrastructure in AWS to deploy {product-title} to AWS GovCloud and hidden regions. See xref:../../registry/configuring_registry_storage/configuring-registry-storage-aws-user-infrastructure.adoc#configuring-registry-storage-aws-user-infrastructure[Configuring the registry for AWS user-provisioned infrastructure].
```

Comment:
Update: See xref:../../registry/configuring_registry_storage/configuring-registry-storage-aws-user-infrastructure.adoc#configuring-registry-storage-aws-user-infrastructure[Configuring the registry for AWS user-provisioned infrastructure] for more information.